### ISSUE TITLE
Properties in BaseDocument of Traversal are empty

### ISSUE THREAD

**Issue by lukasmahr** [arangodb/arangodb-java-driver#50]
Hi,

I think the issue #21 still exists for traversals. I tested the following code from version 2.7.4 up to 3.0.2 with ArangoDB version 3.0.5:

```
TraversalQueryOptions options = new TraversalQueryOptions();
options.setGraphName("GRAPHNAME");
options.setDirection(Direction.OUTBOUND);
options.setStartVertex("COL/123");
TraversalEntity<BaseDocument, BaseDocument> traversalEntity = arangoDb.getTraversal(options, BaseDocument.class, BaseDocument.class);
traversalEntity.getVertices().get(0).getEntity().getProperties().size(); // <--- this returns 0 for every entity
```

The TraversalEntity contains the correct vertices and edges, but the properties are missing.
Reading the startVertex with `arangoDb.getDocument("COL/123", BaseDocument.class)` works fine (including the properties).

Thank you in advance!

**Comment by a-brandt** [comment]
Hello!
I released version 3.0.3 of arangodb-java-driver to fix the problem.
Please try again with that version.

**Comment by lukasmahr** [comment]
Thank you very much! The problem is fixed.

But i recognized another issue:
```
traversalEntity.getPaths().get(0).getEdges().get(0).getFromVertexHandle();
traversalEntity.getPaths().get(0).getEdges().get(0).getToVertexHandle();
```
The properties of the BaseDocument contain the "_from" and "_to" value with the handle. But both of these statements above return NULL.

**Comment by a-brandt** [comment]
I fixed the problem in the current snapshot version 3.0.4-SNAPSHOT and will be released soon.

**Comment by mpv1989** [comment]
fixed in version 3.0.4
